JavaScript/React.md

```diff
@@ -481,10 +481,10 @@ export class ComponentName extends React.Component {
 }
 ```
 
-### Local Component State
+### Local component state
 If MobX is used for state management:
-- prefer using MobX *observable* instead of *this.state* for component's local state; don't use both
-- wrap local state in a single observable object named **componentState**
+- prefer using a MobX *observable* instead of *this.state* for a component's local state; don't use both
```

Comment:
Not sure for the "a MobX" change

Reply:
There should be an article before MobX if the sentence is structured like this - _a_ if _observable_ is not a single item, _the_ if it is a specific one item. If you don't want to treat _observable_ as a noun, I'd recommend writing, for example, prefer using MobX's _observable_ or prefer using _observable_ from MobX or something along those lines